bin/resin-preload

```diff
@@ -97,57 +101,53 @@ const options = {
   apiHost: process.env['API_HOST'],
   registryHost: process.env['REGISTRY_HOST'],
   splashImage: process.env['SPLASH_IMAGE'],
-}
+};
 
 while( argv.length ) {
   switch( argv.shift() ) {
-    case '--app': options.appId = argv.shift(); break
-    case '--img': options.image = argv.shift(); break
-    case '--api-token': options.apiToken = argv.shift(); break
-    case '--api-key': options.apiKey = argv.shift(); break
-    case '--commit': options.commit = argv.shift(); break
-    case '--api-host': options.apiHost = argv.shift(); break
-    case '--registry': options.registryHost = argv.shift(); break
-    case '--splash-image': options.splashImage = argv.shift(); break
+  case '--app': options.appId = argv.shift(); break;
+  case '--img': options.image = argv.shift(); break;
+  case '--api-token': options.apiToken = argv.shift(); break;
+  case '--api-key': options.apiKey = argv.shift(); break;
+  case '--commit': options.commit = argv.shift(); break;
+  case '--api-host': options.apiHost = argv.shift(); break;
+  case '--registry': options.registryHost = argv.shift(); break;
+  case '--splash-image': options.splashImage = argv.shift(); break;
```

Comment:
That was the default rules of eslint, I changed it now. (I also added the missing .eslintrc file).